lib/Runtime/Library/DateImplementation.cpp

```diff
@@ -1306,6 +1307,27 @@ namespace Js {
                     if (lwT >= 60)
                         goto LError;
                     lwTime += lwT;
+                    ss = (ch == ':' || ch == '.') ? (pch++, ssMillisecond) : ssNil;
+                    break;
+                }
+                case ssMillisecond:
+                {
+                    AssertMsg(isNextFieldDateNegativeVersion5 == false, "isNextFieldDateNegativeVersion5 == false");
+
+                    if (lwT >= 0 && lwT < 10)
+                        lwMillisecond = lwT * 100;
```

Comment:
Please always put braces around if-else bodies, even if it's a single statement.